packages/ton-client/src/utils/gestures.ts

```diff
@@ -1,26 +1,30 @@
 export type GestureCallbackFn = (gestureType: string, otherArgs?: any) => void;
 
+export const GESTURE_TYPES = { tap: 'tap', swipe: 'swipe', pinch: 'pinch' };
+
 class GestureHandler {
   TAP_THRESHOLD = 250; // milliseconds
```

Comment:
ooof, I wrote a comment here, too, but I didn't click a button here but directly on review changes in the upper right corner, so this comment was lost -.- 
Let me write it again: 

I think we might want to use `TAP_RADIUS` instead of `TAP_THRESHOLD`: only if a touch moves away at least `TAP_RADIUS` pixels form the initial position it is considered as something else than a tap. 

When using a time based `TAP_THRESHOLD` I can see scenarios in which it fails: 
- e.g. when swiping really fast ('flicking'), e.g. to scroll a large list really quickly, the touch might be shorter then 250 ms and interpreted as a tap. 
- also, when tapping really intently (imagine maybe elders not very familiar with touchscreens), the contact with the screen might be longer than 250 ms and interpreted as a drag or so, while a tap was intended. 

I can implement this, if you want. It's as simple as storing the original position and a boolean `dragging` that get's set to true once the touch moves beyond the `TAP_RADIUS`. If you move back in, dragging remains true, otherwise you'd snap around the initial position when dragging something or similar.

Reply:
changed it, it was also just a first try to make something work